diff --git a/src/core/project-graph/build-dependencies/typescript-import-locator.js b/src/core/project-graph/build-dependencies/typescript-import-locator.js
index 9e778e0e3db507220e8ac34594a7b4fb7243a498..926a20e0826adf6b1735244302132a1d63cfe22d 100644
--- a/src/core/project-graph/build-dependencies/typescript-import-locator.js
+++ b/src/core/project-graph/build-dependencies/typescript-import-locator.js
@@ -14,6 +14,7 @@ class TypeScriptImportLocator {
     fromFile(filePath, visitor) {
         const extension = path.extname(filePath);
         if (extension !== '.ts' &&
+            extension !== '.vue' &&
             extension !== '.tsx' &&
             extension !== '.js' &&
             extension !== '.jsx') {
